opencti-platform/opencti-graphql/src/parser/csv-bundler.ts

```diff
@@ -32,8 +32,10 @@ export const bundleProcess = async (context: AuthContext, user: AuthUser, conten
 
   const bundleBuilder = new BundleBuilder();
   let skipLine = sanitizedMapper.has_header;
-
-  const records = await parsingProcess(content, mapper.separator);
+  let records = await parsingProcess(content, mapper.separator);
+  if (mapper.skipLineChar) {
+    records = records.filter((record) => !record[0].startsWith(mapper.skipLineChar));
+  }
```

Comment:
what if records is not defined ? or record is empty ?
Maybe it could be done when we loop on records below ?

Reply:
I did not try to send an empty CSV files. It worth to try the case.
In my opinion, I prefer to do it before the record because inside the records, we have an option to skip the line for the header. This will help separate the different use cases and improve readability. If we put it inside, we would need to check and manage if we are on the first line of the files or in a comment line, which would bring more complexity. What do you think?